src/nvim/api/private/helpers.c

```diff
@@ -808,6 +811,17 @@ static void init_type_metadata(Dictionary *metadata)
 
   PUT(*metadata, "types", DICTIONARY_OBJ(types));
 }
+static void init_api_level_metadata(Dictionary *metadata)
+{
+  Dictionary version = ARRAY_DICT_INIT;
+  PUT(version, "current", INTEGER_OBJ(NVIM_API_CURRENT));
+  PUT(version, "compatibility", INTEGER_OBJ(NVIM_API_COMPATIBILITY));
+#ifdef NVIM_VERSION_PRERELEASE
+  PUT(version, "prerelease", BOOLEAN_OBJ(true));
```

Comment:
> When a release is made, "pre-release" is disabled.

Can add that step to [release.sh](https://github.com/neovim/neovim/blob/master/scripts/release.sh#L48).